Trello/README.md

```diff
@@ -7,23 +7,27 @@ or customers.
 It needs two software to work:
 
 * [jq](https://stedolan.github.io/jq/) - a JSON parser for the command line
-* [httpie](https://httpie.org) - a command line HTTP client with intuitive UI
+* [cURL](https://curl.se/) - the classic CLI HTTP client
 
-HTTPie can be easly replaced by curl if you prefer (look at the trello_api()
+cURL can be easly replaced by other CLI client (such [HTTPie](https://httpie.org)) if you prefer (look at the trello_api()
 function).
-
 ## Installation
 
 First of all, you need the Trello Key and Trello Token in order to perform API
 calls. Go to the [Trello Developer API Keys page](https://trello.com/app-key/)
-and follow the intructions. Once obtained the two values, edit trello2csv.sh and
+and follow the intructions.
+Once obtained the two values, edit `trello2csv.env` and
 replace the two lines:
 
-    TRELLO_KEY='PUT YOUR KEY HERE'
-    TRELLO_TOKEN='PUT YOUR TOKEN HERE'
+    export TRELLO_KEY='PUT YOUR KEY HERE'
```

Comment:
I still don't understand why we are exporting vars here if we load them from source file...